src/App.js

```diff
@@ -1,200 +1,16 @@
 import React from 'react'
-// import * as BooksAPI from './BooksAPI'
 import './App.css'
+import {Route} from 'react-router-dom'
+import SearchBook from "./SearchBook"
+import HomePage from "./Home"
```

Comment:
Since the component is `HomePage` I'd suggest naming this component `HomePage.jsx`

Reply:
+